modules/apps/object/object-service/service.xml

```diff
@@ -229,12 +229,12 @@
 			<finder-column name="objectDefinitionId" />
 			<finder-column comparator="!=" name="status" />
 		</finder>
-		<finder name="ERC_G_C" return-type="ObjectEntry" unique="true">
```

Comment:
You removed the property `unique="true"` but the return type still `return-type="ObjectEntry"`.

Do you think that there can exist a scenario where this finder can return a list instead of a single Object Entry? 

I think we should reflect on this and maybe change the return type and fix the existing usage in the portal. What do you think?